code/game/shuttles/vox.dm

```diff
@@ -63,6 +71,18 @@ var/global/datum/shuttle/vox/vox_shuttle = new(starting_area=/area/shuttle/vox/s
 	link_to(vox_shuttle)
 	.=..()
 
+	var/datum/holomap_marker/newMarker = new()
+	newMarker.id = "skipjack"
+	newMarker.icon = 'icons/holomap_markers_32x32.dmi'
+	newMarker.filter = HOLOMAP_FILTER_VOX
+	newMarker.x = x
+	newMarker.y = y
+	newMarker.z = z
+	newMarker.pixel_x = -16
```

Comment:
Because that's the value that makes the ship appear aligned with its spawn point on the holomap. It has nothing to do with atom pixel_x/pixel_y since it's a datum, so there's no need to multiply by a world icon multiplier or anything like that.